tests/numpy/test_arithmetic.py

```diff
@@ -123,14 +123,14 @@ def test_matmul_tensor(nps_app_inst):
 
     assert nps_app_inst is not None
 
-    # TODO(bcp): Replace with matmul tests for rank > 2 once implemented.
-    def check_matmul_tensor(_ns_a, _ns_b):
+    # TODO (bcp): Replace with matmul tests for rank > 2 once implemented.
+    def test_matmul_tensor_error(_ns_a, _ns_b):
```

Comment:
Can we revert the prefix of this function back to `check`? The `test` prefix is used to designate test functions to pytest. While this is a nested function, it will improve readability to have the prefix be something other than `test`.